swagger-api-client/publish.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -e
+
+# sbt publish publishes libs to Artifactory for the scala version sbt is running as.
+# sbt +publish publishes libs to Artifactory for all scala versions listed in crossScalaVersions.
+SNAPSHOT=$(( "$TRAVIS_PULL_REQUEST" == "false" && "$TRAVIS_BRANCH" == "develop" ? "SNAP" : "" ))
```

Comment:
How often should this push to artifactory? (Right now it just pushes on every commit for testing purposes.)

Reply:
but then...won't it just make a million of them - one for every commit anyone makes on any branch of this repo?

Reply:
I think wb-libs just does it when there's an open PR, but I'm not sure. In either case: who cares :)